test/support/exercise_test_case.ex

```diff
@@ -97,6 +99,8 @@ defmodule ElixirAnalyzer.ExerciseTestCase do
           comments =
             result.comments
             |> Enum.map(fn comment_details -> comment_details.comment end)
+            # There are too many compiler warnings in tests
```

Comment:
Good choice. We wouldn't want to make all test snippets perfect, and we also don't need to do any assertions about when which warning is produced because that's Elixir internals.

Reply:
Yeah, that would be so much work. Not worth it ^^